lib/galaxy/tools/data_fetch.py

```diff
@@ -352,6 +352,7 @@ def _has_src_to_path(upload_config, item, is_dataset=False):
             name = "Pasted Entry"
     else:
         assert src == "path"
+        print(item)
```

Comment:
Want to remove this I assume.

Reply:
Oops, yeah, thanks, I've rebased this out.